### PR TITLE
Disable ipv6 flag

### PR DESCRIPTION
**Pull request by enriavil1**
This was a great idea #6261 
Added two new flags to turn off downloader ipv6 and/or ipv4 usage

### CODE REVIEW

File: cmd/utils/flags.go [hunk 1]
```diff
@@ -723,6 +723,12 @@ var (
 		Usage: "Port for sentinel",
 		Value: 7777,
 	}
+
+	DisableIPV6 = cli.BoolFlag{
+		Name:  "disable.ipv6",
```
Comment (AlexeyAkhunov) [inline cmd/utils/flags.go:728]: Name of the flag needs to mention `downloader.` because it is setting for that component only

**Review by AskAlexSharov — CHANGES_REQUESTED** [review]
need same flag for ipv4

File: turbo/cli/flags.go [hunk 2]
```diff
@@ -250,6 +250,10 @@ func ApplyFlagsForEthConfig(ctx *cli.Context, cfg *ethconfig.Config) {
 		}
 	}
 
+	if ctx.Bool(utils.DisableIPV6.Name) {
```
Comment (AskAlexSharov) [inline turbo/cli/flags.go:253]: ./cmd/downloader 
also need this flag

**Review by AskAlexSharov — CHANGES_REQUESTED** [review]

File: turbo/cli/flags.go [hunk 3]
```diff
@@ -250,6 +250,15 @@ func ApplyFlagsForEthConfig(ctx *cli.Context, cfg *ethconfig.Config) {
...
+		log.Info("Downloader Disabled IPV6")
+		cfg.Downloader.ClientConfig.DisableIPv6 = true
+	}
+
+	if ctx.Bool(utils.DisableIPV4.Name) {
+		log.Info("Downloader Disabled IPV4")
+		cfg.Downloader.ClientConfig.DisableIPv4 = false
```
Comment (AskAlexSharov) [inline turbo/cli/flags.go:260]: 1. probably here must be `true`
2. let's write exactly same style log line as we do in ./cmd/downloader and in ./cmd/erigon (about downloader component configuration): 
1-line which describe all params: ipv4,ipv6,uploadrate,downloadrate,...

**Review by AskAlexSharov — CHANGES_REQUESTED** [review]

File: turbo/cli/flags.go (hunk 3 above)
Comment (enriavil1) [reply turbo/cli/flags.go]: Done!

**Review by AskAlexSharov — APPROVED** [review]

**Pull request merged**